### DISCUSSION TITLE
GPT-4 Turbo  completion tokens error

### DISCUSSION THREAD

**Discussion by FamedBear16** [logancyang/obsidian-copilot#207] [Q&A]
Hi,
I am using Copilot in Conversation mode, and set max tokens to 8,000.  I am submitting a question + context of around 7900 tokens. 
I am getting the following error:
![image](https://github.com/logancyang/obsidian-copilot/assets/119869050/6b62dabf-f766-4618-89aa-9379523ad6c3)

However if I use GPT 3.5 16K all work well?
What is happening?
Chat GPT 4.0 has a much larger context.

I did another test
If I set the limit to 4000 Tokens, then GPT 4.0 Turbo, with work well with the same input of around 7900 tokens

Regards,

Pierpaolo

**Comment by logancyang** [comment]
Hi @FamedBear16, sorry for the late reply, can you refer to https://github.com/logancyang/obsidian-copilot/issues/256#issuecomment-1909187362
